test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1

```diff
@@ -228,7 +228,7 @@ Describe "PowerShellGet - Script tests (Admin)" -tags @('Feature', 'RequireAdmin
         Remove-InstalledScripts
     }
 
-    ## Marked as 'Pending' on Linux for now because the test requires root privilege but we cannot do it now in our Travis CI Linux build
+    ## Marked as 'Pending' on Linux for now because the test requires root privilege but we cannot do it now in our Azure DevOps Linux build
```

Comment:
Is this true in Azure DevOps? Should we try enabling these two tests?

Reply:
We can try

Reply:
I’ve temporarily made it run on linux so we can check.